src/main/scad/laszlo-molnar/l-i-vator-ii/molnar.l-i-vator-ii.scad

```diff
@@ -0,0 +1,82 @@
+include <puzzlecad.scad>
+
+box_puzzle_border = 6;
+
+$burr_inset = 0.125;
+$burr_bevel = 1;
+$unit_beveled = true;
+$burr_scale = 17;
+
+dim = $burr_scale * 3 + box_puzzle_border * 2 + $burr_inset * 2;
+height = $burr_scale * 3 + box_puzzle_border + $burr_inset * 2;
+
+mid = box_puzzle_border / 2 + 0.5;
+far = dim - mid;
+    
+*box();
+*pieces();
+*cap();
+
+module box() {
+    render(convexity = 2)
+    difference() {
+        beveled_cube([dim, dim, height]);
```

Comment:
Add $burr_bevel = 0.5 here for consistency? (I don't have a super strong opinion, more of a question)